functions/jens.js

```diff
@@ -49,7 +49,8 @@ const makeSendMessage = (sendData) => {
 Route: ${sendData.zlaggableName}
 Crag: ${sendData.cragName}
 Area: ${sendData.areaName}
-Difficulty: ${sendData.difficulty}`
+Difficulty: ${sendData.difficulty}
+Comment: ${sendData.comment.match(/(.|[\r\n]){1,100}/g).join("\n")}`;
```

Comment:
```suggestion
Comment: ${sendData.comment}`;
```

Reply:
If you do this change I'll accept and deploy the comment and we can test by changing the number of sends of a user in the DB